modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentsSectionDisplayContext.java

```diff
@@ -30,15 +36,16 @@ public ContentsSectionDisplayContext(
 		GroupLocalService groupLocalService,
 		HttpServletRequest httpServletRequest, Language language,
 		ObjectDefinitionService objectDefinitionService,
-		ObjectDefinitionSettingLocalService
-			objectDefinitionSettingLocalService) {
+		ObjectDefinitionSettingLocalService objectDefinitionSettingLocalService,
+		Portal portal) {
```

Comment:
Can you please add `portal` to `BaseSectionDisplayContext` as it used by both subclasses?

Reply:
Forget about it. I've realized that there is a third subclass not using Portal